Makefile

```diff
@@ -55,3 +55,10 @@ generate:
 docs: moactl
 	rm -rf docs && mkdir docs
 	./moactl docs -d ./docs -f markdown
+
+mocks:
```

Comment:
I think it might make sense for `mocks` to be a dependency of `test`:
```diff
diff --git a/Makefile b/Makefile
index 9402f5e..f538192 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ moactl: generate
        go build .
 
 .PHONY: test
-test:
+test: mocks
        ginkgo -r cmd pkg
 
 .PHONY: fmt
```

Reply:
you need to run this only if you change a underlying interface, so I think it's better to not run it on every make to have it more solid.

Reply:
makes sense